gateway/gateway.go

```diff
@@ -163,7 +163,7 @@ func (gateway *HandleT) initUserWebRequestWorkers() {
 			batchRequestQ:  make(chan *batchWebRequestT),
 			reponseQ:       make(chan map[uuid.UUID]string),
 			workerID:       i,
-			batchTimeStat:  gateway.stats.NewStat("gateway.batch_time", stats.TimerType),
+			batchTimeStat:  gateway.stats.NewStat(fmt.Sprintf("gateway.batch_time_%d", i), stats.TimerType),
```

Comment:
Let's use  tagged stat with workedId tag?
Also why do we need workerId?

Reply:
done